build.ps1

```diff
@@ -64,6 +64,11 @@ function ClearBaselineFiles($root)
 
 ################################################# Functions ############################################################
 
+if ($env:VSINSTALLDIR)
+{
+    $msbuildDir = $env:VSINSTALLDIR+"\MSBuild\Current\Bin\amd64";
```

Comment:
Not sure why we are appending "amd64" here.
$msbuildDir is defined without it:
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/5d1bee56a0612d5187eba7ddb3edf11f05f8d589/build.ps1#L4

Reply:
@GeoK I took the amd64 off and performed a check if the folder exists.